tests/unit/resolution_resolvelib/conftest.py

```diff
@@ -50,16 +48,10 @@ def preparer(finder):
 
 @pytest.fixture
 def factory(finder, preparer):
-    make_install_req = partial(
-        install_req_from_req_string,
-        isolated=False,
-        wheel_cache=None,
-        use_pep517=None,
-    )
     yield Factory(
         finder=finder,
         preparer=preparer,
-        make_install_req=make_install_req,
+        make_install_req=install_req_from_line,
```

Comment:
Was there any particular reason for this change? I'm not sure I fully understand the differences between `install_req_from_line` and `install_req_from_req_string`. I don't think it makes any difference in practice, though.

Reply:
I changed it simply for convinience, because some of the tests need to use `install_req_from_line`. This seems to accept more input than `install_req_from_req_string`, but I admit I’m not entirely sure of the differnce either.

Reply:
Cool, that's a good enough reason for me 🙂